src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -373,23 +454,37 @@ public async Task<ApplicationsViewModel> GetApplications(ApplicationsRequest req
         {
             var result = await _pledgeService.GetApplications(request.AccountId, request.PledgeId);
 
+            var pledgeApplications = await _pledgeService.GetApplications(request.AccountId, request.PledgeId);
```

Comment:
Struggling to fathom this tbh - aren't we making two identical calls to the api here?

Reply:
that must have been a code refactoring to linq from a foreach that Resharper handles and I missed it